KEYS

```diff
@@ -409,61 +409,61 @@ F57OPCcdXr6r1SO82p7ahP/gAwiVljbptDPcLgKFbm3d3OyvwHCJ4ei/PK/iHA==
 =0R0k
 -----END PGP PUBLIC KEY BLOCK-----
 
-pub   rsa4096 2024-12-20 [SC]
-      82173B76B344A75DBE703159E8F710CEB390075B
-uid           [ultimate] Josh Adams <jadams@apache.org>
-sig 3        E8F710CEB390075B 2024-12-20  [self-signature]
-sub   rsa4096 2024-12-20 [E]
-sig          E8F710CEB390075B 2024-12-20  [self-signature]
+pub   rsa4096 2025-01-13 [SC]
+      03CAD0116AFD4359769CDA907B3C84C0C2319C12
+uid           [ultimate] Joshua Adams <jadams@apache.org>
+sig 3        7B3C84C0C2319C12 2025-01-13  [self-signature]
+sub   rsa4096 2025-01-13 [E]
+sig          7B3C84C0C2319C12 2025-01-13  [self-signature]
 
```

Comment:
The digest algorithm for your new key is sha256. Although that's allowed, infra recommends we use of sha512. Can you create a new key with sha512? The KEYS file mentions the gpg configuration for that.

Reply:
Will do.  Somehow I missed that.